projects/bp-strapi/unit-tests/mergeTags.test.js

```diff
@@ -0,0 +1,81 @@
+const { setupTestData, cleanUpTestData } = require("./helpers/setupTestData");
+const {
+  mergeSourceTagIntoTargetTag,
+} = require("../src/api/custom-tag-resolver");
+
+const TAG_TYPES = [
+  "location-tag",
+  "person-tag",
+  "keyword-tag"
+]
+
+describe("Merge different tag types", () => {
+  beforeAll(async () => {
+    await setupTestData();
+  });
+
+  afterAll(async () => {
+    await cleanUpTestData();
+  });
+
+  TAG_TYPES.map(tagType => describe(`${tagType}s`, () => {
+    const tagQuery = strapi.db.query(`api::${tagType}.${tagType}`);
+    const tagService = strapi.service(`api::${tagType}.${tagType}`);
+
+    beforeAll(async () => {
+      await tagService.create({
+        data: {
+          name: `${tagType} 1`,
+          pictures: [1, 2],
+          verified_pictures: [3, 4, 5],
```

Comment:
Do we know for sure that the pictures created in the test data will always have the ids 1-10?

Reply:
Somehow it works, just trust the documentation and test setup i guess. Locally it always worked, no matter how many times we retried the tests (watch mode and non-watch mode)